android/java/proguard.flags

```diff
@@ -102,3 +102,6 @@
     *** dismissActivityIfNeeded(...);
 }
 
+-keep class org.chromium.chrome.browser.tab_group_sync.StartupHelper {
```

Comment:
We should move it to `android/java/apk_for_test.flags`, here we add rules only for calls via reflection.

Reply:
thanks, updated https://github.com/brave/brave-core/pull/29201/commits/3af4a8b854a62428ccef337a78a0e10f1333d652